### PR TITLE
Cr 825

### PR DESCRIPTION
**Pull request by andrewjohnysons**
**Motivation and Context**
Add tests for Launch Survey on Telephone Capture Endpoint

**What has changed**
TestCaseEndpoints now has new methods to test the Launch Survey process.
testTelephoneEndpoints.feature has two more steps:-
    When CC Advisor selects the survey launch
    Then a Survey Launched event is emitted to RM, which contains the launch status type

**How to test?**
mvn clean install will run the cucumber tests which should run successfully.

### CODE REVIEW

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java [hunk 1]
```diff
@@ -797,4 +799,70 @@ public void cc_Advisor_selects_the(String statusSelected) {
...
+        queueName,
+        clazzName,
+        timeout);
+
+    SurveyLaunchedEvent launchedEvent = null;
+    try {
+      launchedEvent =
+          rabbit.getMessage(
+              queueName, SurveyLaunchedEvent.class, TimeoutParser.parseTimeoutString(timeout));
+    } catch (Exception exception) {
```
Comment (PeterBochel) [inline src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java:859]: Surely easier to not catch the exception and let the test fail

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java [hunk 2]
```diff
@@ -797,4 +799,70 @@ public void cc_Advisor_selects_the(String statusSelected) {
...
+
+  private String requestSurveyLaunch() {
+    log.with(telephoneEndpointUrl).info("The url for requesting the survey launch");
+
+    String launchSurveyResponseString = null;
+    try {
+      launchSurveyResponseString =
+          getRestTemplate().getForObject(telephoneEndpointUrl, String.class);
+    } catch (Exception ex) {
+      System.out.println(ex.getMessage());
```
Comment (PeterBochel) [inline src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java:833]: Why not just let the test fail?

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java (hunk 1 above)
Comment (andrewjohnysons) [reply src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java:852]: Removed try catch

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java (hunk 2 above)
Comment (andrewjohnysons) [reply src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java]: Added
      fail("SURVEY launch HAS FAILED - the contact centre does not give a response code of 200");

**Pull request merged**